tests/karma/unit/services/download-url.js

```diff
@@ -50,12 +50,12 @@ describe('DownloadUrl service', function() {
     });
   });
 
-  it('builds url for forms', function() {
+  it('builds url for reports', function() {
     Language.returns(Promise.resolve('en'));
     GenerateLuceneQuery.returns({ query: 'form:P' });
```

Comment:
GenerateLuceneQuery and Language should not be called any more. Consider asserting that they're not?

Reply:
Done